include/kernels/coarse_op_kernel.cuh

```diff
@@ -120,74 +128,176 @@ namespace quda {
     static_assert(max_color_height_per_block % tile_height_vuv == 0, "max_color_height_per_block must be divisible by tile height");
     static_assert(max_color_width_per_block % tile_width_vuv == 0, "max_color_width_per_block must be divisible by tile width");
 
-    CalculateYArg(coarseGauge &Y, coarseGauge &X, 
+    /**
+       @brief Constructor for CalculateYArg
+       @param[out] Y Accessor to the coarse gauge field
+       @param[out] X Accessor to the coarse clover field
+       @param[in,out] Y Atomic accessor to the coarse gauge field
+       @param[in,out] X Atomic accessor to the coarse clover field
+       @param[in,out] UV Accessor to the uv temp packed colorspinor field
+       @param[in,out] AV Accessor to the av temp packed colorspinor field
+       @param[in] U Accessor to the fine gauge field
+       @param[in] V Accessor to the packed nullspace colorspinor field
+       @param[in] C Accessor to the fine clover field
+       @param[in] Cinv Accessor to the fine inverse clover field
+       @param[in] kappa Kappa parameter from the fine operator
+       @param[in] mass Mass parameter from the fine operator
+       @param[in] mu Twisted mass parameter from the fine operator
+       @param[in] mu_factor Additional twisted mass parameter for coarse-grid stability
+       @param[in] x_size_ Fine-grid geometric dimensions
+       @param[in] xc_size_ Coarse-grid geometric dimensions
+       @param[in] Pointer to fine-to-coarse look-up table (memory space is same compute)
```

Comment:
Add `fine_to_coarse`, `coarse_to_fine`, `bidrectional` variable names to the doxygen

Reply:
c94109308c46ca051eea11a865a2f445d990e257